packages/jsoo-react/jsoo-react.0.1/opam

```diff
@@ -32,7 +32,6 @@ build: [
     "-j"
     jobs
     "@install"
-    "@runtest" {with-test}
```

Comment:
I would revert this. The failure to install conf-npm in the CI is not preventing the merge and having the tests run in all other environments will ensure that this package is properly tested each time an update of its dependencies is released

Reply:
The removal was not due to the conf-npm error, but a different one:

```
Error: Cannot find module 'react-dom/test-utils'
```

Can be seen [here](https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/8cf8a29a6bbd5fe9a3629a0852ad091c87849542/variant/compilers,4.13,jsoo-react.0.1,tests).

The reason is that to run the tests we need to install some dependencies from npm first. I guess we should add `conf-npm` as a `with-test` dependency, and then somehow make sure that `npm install` is called before or as part of the `runtest` alias. I am not sure how to do this last part, so for now I thought I would disable the tests as a workaround while I figure it out.

Reply:
Oh, I see. Thanks for the clarification. In that case, unless you can bundle the dependencies already in the tarball, let's disable it.